Server/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+
+const Camps = require('../database-mongodb/Camp.js')
+
+app.get('/:campsiteId', (req, res) => {
+  console.log(req.params)
+    Camps.find({ newId: req.params.campsiteId}, (err, camps)=> {
+        if (err){
+          res.send(err)
```

Comment:
@alefranco1 @bastipacudan Nice job on doing something with the error :) Be sure to send it with the appropriate status code.